Index: pyHesiodFS.py
===================================================================
--- pyhesiodfs/pyHesiodFS.py	(revision 25441)
+++ pyhesiodfs/pyHesiodFS.py	(working copy)
@@ -176,6 +176,8 @@
             if len(filsys.filsys) >= 1:
                 pointers = filsys.filsys
                 pointer = pointers[0]
+                if pointer['type'] == 'ERR':
+                    return None
                 if pointer['type'] != 'AFS' and pointer['type'] != 'LOC':
                     syslog(LOG_NOTICE, "Unknown locker type "+pointer['type']+" for locker "+name+" ("+repr(pointer)+" )")
                     return None
